plugins/org.komodo.core/src/org/komodo/repository/ObjectImpl.java

```diff
@@ -68,6 +71,87 @@ ObjectImpl accessOuter() {
     }
 
     /**
+     * Implementation specific and not to be included on the interface!
+     *
+     * @param uow transaction for this operation. Can be null.
+     *
+     * @return the underlying object of this {@link ObjectImpl}
+     *
+     * @throws KException if an error occurs
+     */
+    public Node node(UnitOfWork uow) throws KException {
```

Comment:
Figured as long as its not on the interface then effort really has to be put in to cast to ObjectImpl and then call it. 
I use it in a unit test in TestLocalRepostiory to check that a real jcr node exists. The test kept producing a PathNotFoundException so wanted to do assertNotNull(kObject.node()). If we have an alternative that would achieve the same thing?